src/main/com/smithsmodding/armory/client/textures/MaterializedTextureCreator.java

```diff
@@ -60,31 +55,7 @@
 
     //Initializes the dummy gui material with a proper set of render info.
     static {
-        guiMaterial = new MedievalCoreArmorMaterial("", "", "", 0F,0F,0,0,0F) {
-            /**
-             * Method to get the BaseDurability of a piece of armor made out of this material.
-             *
-             * @param armor The armor to get the base durability for.
-             * @return The durability of a piece of armor made out of this material.
-             */
-            @Nonnull
-            @Override
-            public Integer getBaseDurabilityForArmor(@Nonnull IMultiComponentArmor armor) {
-                return 0;
-            }
-
-            /**
-             * Method to get all the default capabilities this ArmorMaterial provides.
-             *
-             * @param armor
-             * @return All the default capabilities this ArmorMaterial provides.
-             */
-            @Nonnull
-            @Override
-            public HashMap<Capability<? extends IArmorCapability>, Object> getOverrideCoreMaterialCapabilities(IMultiComponentArmor armor) {
-                return new HashMap<>();
-            }
-        };
+        guiMaterial = new MedievalCoreArmorMaterial("", "", "", 0F, 0F, 0, 0, 0F, new ArmorCapabilityManager());
         guiMaterial.setRenderInfo(new MaterialRenderControllers.AbstractMaterialTextureController() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)